Automattic/wp-calypso
# Analytics: remove login loading tracking

This pull request reverts #21393 and #21965 after we've collected sufficient data
  
#### Testing instructions
  
1. Run `git checkout remove/loading-tracking` and start your server, or open a [live branch](https://calypso.live/?branch=remove/loading-tracking)
2. Open the [`Login` page](http://calypso.localhost:3000/log-in)
3. Check that `calypso_load_start` and `calypso_load_end` are not emitted.
